spec/refile/attachment_spec.rb

```diff
@@ -167,7 +167,7 @@
         it "handles redirect loops by trowing errors" do
           expect do
             instance.remote_document_url = "http://www.example.com/loop"
-          end.to raise_error(RestClient::MaxRedirectsReached)
+          end.to raise_error(RestClient::ExceptionWithResponse)
```

Comment:
If you happen to have any code that depends on MaxRedirectsReached, this will break it (caused by the new rest-client version), there doesn't seem to be any code that uses it though.

Reply:
no, not that I know 👍